### ISSUE TITLE
Add positions to badges

### ISSUE THREAD

**Issue by allanberger** [sido378/selfie_bellen#5]
Position options: bottom left (bl), bottom right (br)

bottom left:
bottom = image_height - badge_height
left = 0

bottom right:
bottom = image_height - badge_height
right = image_width - badge_width

**Comment by allanberger** [comment]
Code for botto left positioning of badge:
    # pasting badge on bottom left edge
    img.paste(badge, (0, img_h - badge_h), badge)

**Comment by allanberger** [comment]
dictionary for badges with positions @sido378:

```json

TEMPLATES = {
    "entries": {
        "template": "vdb_badge1",
        "position": "right"
    },
    {
        "template": "vdb_badge2",
        "position": "right"
    },
    {
        "template": "vdb_badge3",
        "position": "right"
    },
    {
        "template": "vdb_badge4",
        "position": "left"
    },
    {
        "template": "vdb_badge5",
        "position": "left"
    }
}```

**Comment by allanberger** [comment]
jfyi @sido378: the branch for this code is `align-badges`

**Comment by allanberger** [comment]
Should be done with #14.

**Comment by allanberger** [comment]
Closed with #14.
